CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Implemented service and repository design patterns for retrieving data from AcademiesDb.
 - Data source information cached to increase performance of all Trust pages.
 - Trust summary information (UID, name, type and number of academies) cached to improve performance of all Trust pages.
+- Feature flags added to allow for selectively showing features in different environments
```

Comment:
Under the (keep a changelog)[https://keepachangelog.com/en/1.0.0/]  guidance we're using, this would belong to an `Added` section which should be above the `Changed` section